mailet/standard/src/main/java/org/apache/james/transport/mailets/Null.java

```diff
@@ -26,25 +26,15 @@
 
 /**
  * Simplest Mailet which destroys any incoming messages.
- *
  */
 public class Null extends GenericMailet {
 
-    /**
-     * Set this mail to GHOST state, indicating that no further processing 
-     * should take place.
```

Comment:
I though it says the same as "Simplest Mailet which destroys any incoming messages.", it's why I removed it. Do you still think it's useful ?